config.yaml

```diff
@@ -5,10 +5,49 @@
 
 options:
   # An example config option to customise the log level of the workload
-  log-level:
+  customer:
     description: |
-      Configures the log level of gunicorn. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
-    default: "info"
+      Name of the customer that will be added as label to collected metrics. It's useful when
+      Prometheus collects metrics from multiple organizations/customers.
+    default: ""
+    type: string
+  cloud-name:
+    description: |
+      Arbitrary name for a cloud/deployment. This name will be added as label to collected
+      metrics. It's useful when Prometheus collects metrics from multiple controllers.
+    default: ""
+    type: string
+  controller-url:
+    description: Endpoint of a juju controller in format <IP>:<PORT>
+    default: ""
+    type: string
+  controller-ca:
```

Comment:
I think that in programming context `ca` and `ca_cert` are generally interchangeable terms. People understand that when you use `ca` in code, you refer to (root) CA certificate, rather that the Certificate Authority organization itself.

For example, Vault charm also uses such shortened terms:
* [`ssl-ca` config option](https://charmhub.io/vault/configure#ssl-ca) which refers to root CA certificate
* [`generate-root-ca` action](https://charmhub.io/vault/actions#generate-root-ca) that generates CA certificate

Reply:
I will fight for it if I am the code reviewer of charm-vault.  I do not find similar usage in upstream [vault](https://github.com/hashicorp/vault).

generate-root-ca means
      Genearte ca private keys + certificate, so that vault can perform full functionality of Root Certificate Authority. Not only "certificate" as I understand

ssl-ca
     For me... it's quite confuse if no document. Can I get the root CA private key or the certificate?

If we can add 4 more character to avoid confusion,  I prefer we should do it.

Reply:
config option renamed